src/js/components/ToggleButtonGroup/__tests__/ToggleButtonGroup-test.tsx

```diff
@@ -0,0 +1,490 @@
+import React from 'react';
+import { fireEvent, render, screen } from '@testing-library/react';
+import { axe } from 'jest-axe';
+import userEvent from '@testing-library/user-event';
+
+import 'jest-styled-components';
+import 'jest-axe/extend-expect';
+import 'regenerator-runtime/runtime';
+import '@testing-library/jest-dom';
+
+import { Grommet } from '../../Grommet';
+import { Add } from 'grommet-icons';
+import { ToggleButtonGroup } from '..';
+
+describe('ToggleButtonGroup', () => {
+  test('should have no accessibility violations', async () => {
+    const { container } = render(
+      <Grommet>
+        <ToggleButtonGroup options={['Option 1', 'Option 2', 'Option 3']} />
+      </Grommet>,
+    );
+    const results = await axe(container);
+    expect(results).toHaveNoViolations();
+    expect(container.firstChild).toMatchSnapshot();
+  });
+
+  test('string options', () => {
+    const { container } = render(
+      <Grommet>
+        <ToggleButtonGroup options={['one', 'two']} />
+      </Grommet>,
+    );
+
+    expect(container.firstChild).toMatchSnapshot();
+  });
+
+  test('object options', () => {
+    const { container } = render(
```

Comment:
```suggestion
    const { asFragment } = render(
```
We should use `asFragment` instead of `container` now.

Reply:
done